host/ProtocolGateway.Host.Fabric.FrontEnd/ProtocolGateway.Host.Fabric.FrontEnd.csproj

```diff
@@ -0,0 +1,142 @@
+      <Private>True</Private>
+    </Reference>
+    <Reference Include="Microsoft.ServiceFabric.Internal.Strings, Version=5.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=AMD64">
+      <HintPath>..\..\packages\Microsoft.ServiceFabric.5.1.163\lib\net45\Microsoft.ServiceFabric.Internal.Strings.dll</HintPath>
+      <Private>True</Private>
+    </Reference>
+    <Reference Include="Microsoft.ServiceFabric.Services, Version=5.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=AMD64">
+      <HintPath>..\..\packages\Microsoft.ServiceFabric.Services.2.1.163\lib\net45\Microsoft.ServiceFabric.Services.dll</HintPath>
+      <Private>True</Private>
+    </Reference>
+    <Reference Include="ServiceFabricServiceModel, Version=5.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=AMD64">
+      <HintPath>..\..\packages\Microsoft.ServiceFabric.Services.2.1.163\lib\net45\ServiceFabricServiceModel.dll</HintPath>
+      <Private>True</Private>
+    </Reference>
+    <Reference Include="System" />
+    <Reference Include="System.Collections.Immutable, Version=1.2.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL">
+      <HintPath>..\..\packages\System.Collections.Immutable.1.2.0\lib\portable-net45+win8+wp8+wpa81\System.Collections.Immutable.dll</HintPath>
+      <Private>True</Private>
+    </Reference>
+    <Reference Include="System.Core" />
+    <Reference Include="System.Fabric, Version=5.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=AMD64">
+      <HintPath>..\..\packages\Microsoft.ServiceFabric.5.1.163\lib\net45\System.Fabric.dll</HintPath>
+      <Private>True</Private>
+    </Reference>
+    <Reference Include="System.Runtime.Serialization" />
+  </ItemGroup>
+  <ItemGroup>
+    <Compile Include="configuration\AzureSessionStateConfiguration.cs" />
+    <Compile Include="configuration\GatewayConfiguration.cs" />
+    <Compile Include="configuration\IoTHubConfiguration.cs" />
+    <Compile Include="configuration\MqttServiceConfiguration.cs" />
+    <Compile Include="configuration\ServiceFabricConfigurationProvider.cs" />
+    <Compile Include="logging\Logger.cs" />
+    <Compile Include="logging\ServiceLogger.cs" />
+    <Compile Include="mqtt\MqttCommunicationListener.cs" />
+    <Compile Include="Properties\AssemblyInfo.cs" />
+    <Compile Include="FrontEnd.cs" />
+    <Compile Include="Program.cs" />
+  </ItemGroup>
+  <ItemGroup>
+    <None Include="PackageRoot\Config\FrontEnd.AzureState.json" />
+    <None Include="PackageRoot\Config\FrontEnd.IoTHubClient.json" />
+    <None Include="PackageRoot\Config\FrontEnd.json" />
+    <None Include="PackageRoot\Config\FrontEnd.Mqtt.json" />
+    <None Include="PackageRoot\Config\Settings.xml" />
+    <None Include="PackageRoot\Data\protocol-gateway.contoso.com.pfx" />
```

Comment:
><None Include="PackageRoot\Data\protocol-gateway.contoso.com.pfx" /> [](start = 4, length = 68)

can you use the file under /host/shared instead of copying it?